src/nvlink/worker_32bit.py

```
@@ -0,0 +1,377 @@
+
+        start_time = time.time()
+        while True:
+            try:
+                status = self.nvlink.NVStatus()
+                if status < 0:
+                    # Error
+                    return False
+                if status >= download_count:
+                    # Download complete
+                    return True
+            except:
```

Ruff (0.14.14)

[error] Do not use bare `except`

(E722)